etc/gatekeeper.yaml

```diff
@@ -0,0 +1,27 @@
+# Logging settings.
+log:
+  # The desired logging level.
+  # Allowed values are "debug", "info", "warn", "error", "panic" and "fatal"
+  level: debug
+
+# Settings for Ethereum keys
+ethereum:
+  # path to keystore
+  key_store: "./keys/state/gate/k2/"
+  # passphrase for keystore
+  pass_phrase: "111111111"
+
+blockchain:
+  # endpoint: "https://rinkeby.infura.io/00iTrs5PIy0uGODwcsrb"
+  # sidechain_endpoint: "https://sidechain-dev.sonm.com"
+
+gatekeeper:
+  # Define direction .
+  # Allowed values are "debug", "info"
```

Comment:
debug direction and info direction? be careful with copypaste-driven-development